client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -60,6 +60,18 @@ const formatIfValid = (
   }
 ): string => (isValid(date) ? format(date, dateFormat, options) : '');
 
+// Adds the current time to a date object (that presumably has 00:00 as its time
+// component)
+const addCurrentTime = (date: Date | null) => {
+  if (date === null) return date;
+  const d = new Date();
+  const e = new Date(d);
+  const msSinceMidnight = d.getTime() - e.setHours(0, 0, 0, 0);
+  const newDate = new Date(date);
+  newDate.setTime(date.getTime() + msSinceMidnight);
```

Comment:
```suggestion
  newDate.setTime(newDate.setHours(0, 0, 0, 0) + msSinceMidnight);
```

Would this work as well? then you don't need the assumption that the input date is at 00:00

Reply:
Yeah, sure. Have actually just used `e`, here since it's a new Date that has already had it's time set to 0: 81262a2af